src/containers/cyclicbuffer.d

```diff
@@ -0,0 +1,828 @@
+/**
+ * Cyclic Buffer
+ * Copyright: © 2016 Economic Modeling Specialists, Intl.
```

Comment:
I'm no expert but I think it's normal practice to assign copyright in cases like these where you wish to get your code into someone else's project.  That way we have the right to distribute it under the license for this project.